Supplementing Polish translation.

PaweX opened the pull request:
Supplementing the missing translation in Polish language and some little corrections&fixes.

PaweX committed:
Merge pull request #1 from jsettlers/master

Update from newest jsettlers code

PaweX committed:
Supplementing the missing translation

Supplementing the missing translation and some little corrections&fixes

PaweX commented on jsettlers.graphics/src/main/resources/jsettlers/graphics/localization/labels_pl.properties:
```diff
@@ -298,8 +339,69 @@ bad-folders-version = Potrzebujesz plików graficznych gry Settlers III Złota E
+
+join-game-panel-number-of-players = Liczba graczy
+join-game-panel-start-resources = Materiały początkowe
+join-game-panel-new-single-player-game-title = Nowa Gra - Jeden Gracz
+join-game-panel-new-multi-player-game-title = Nowa Gra Sieciowa
+join-game-panel-join-multi-player-game-title = Dołącz do Gry Sieciowej
```
It looks like the game doesn't use them currently. But it is a bug. At least in version 88ba0d98

paulwedeck replied:
They are used correctly.
You can see them as title when you go to "Start Single Player Game", "New Network Game" or "Join Network Game" (and select a map).

PaweX replied:
Ok, yes you right (I didn't notice they are being used), but see this:

Game uses here **start-newmultiplayer-start** instead of **join-game-panel-new-multi-player-game-title** (or equivalent):
![obraz](https://user-images.githubusercontent.com/3579486/79911247-5b12cc80-8420-11ea-93d2-103da4746289.png)

and here **start-joinmultiplayer-start** instead of **join-game-panel-join-multi-player-game-title** (or equivalent):
![obraz](https://user-images.githubusercontent.com/3579486/79911793-42ef7d00-8421-11ea-9d23-b8fc6a3aa17e.png)

paulwedeck replied:
Because this panel is the start-newmultiplayer panel and apparently all labels specific to a panel start with their panel name.

PaweX replied:
So it means that **start-newmultiplayer-title** and **start-joinmultiplayer-title** are missing.

paulwedeck replied:
No it just has another name. If it bothers you that much you could just change all occurrences of start-newmultiplayer-start to start-newmultiplayer-title (most importantly in MainMenuPanel.java)

PaweX replied:
I thought start-newmultiplayer-start and start-joinmultiplayer-start are used for some buttons. In English file, start-joinmultiplayer-start is translated to "Join". So do you say they are not used for any buttons?

paulwedeck replied:
Apparently, yes. Look at MainMenuPanel

PaweX replied:
Ok, I may have a look tomorrow.

PaweX replied:
But if I change that, I will create a new pull request to keep order. But this one here will need to be merged first. Are you able to apply it?

paulwedeck replied:
No. Only @andreas-eberle and @tom-pratt can and both haven't shown up in months

PaweX replied:
:/ , So there is no point to add new pull requests as there is already a few not merged yet. After they are merged, my code will be not up to date.

PaweX replied:
Naming and using the name labels is not consistent in MainMenuPanel.java. Some of them are duplicated and some of them are not used.

I focus mainly at sections: _# Start screen_ and _# Swing extras_ in lang files.
Like there is:
```
# Start screen
start-newgame = New Single Player Game
```
and
```
# Swing extras
main-panel-new-single-player-game-button = Start Single Player Game
```

Why somebody added this? _main-panel-new-single-player-game-button_ is long but there was already _start-newgame_. But main menu uses the first one. And then there is
`start-newmultiplayer = New Network Game`
So why not _main-panel-new-multi-player-game-button_? Or why then _main-panel-new-single-player-game-button_ is used instead of _start-newgame_?
But here anyway "the best" is settings button:
`settings-title = Settings`

So all of them in the code look like this:
```
		registerMenu("main-panel-new-single-player-game-button", e -> setCenter("main-panel-new-single-player-game-button", openSinglePlayerPanel));
		registerMenu("start-loadgame", e -> setCenter("start-loadgame", openSaveGamePanel));
		registerMenu("settings-title", e -> {
			setCenter("settings-title", settingsPanel);
			settingsPanel.initializeValues();
		});
		registerMenu("start-newmultiplayer", e -> setCenter("start-newmultiplayer-start", newMultiPlayerGamePanel));
		registerMenu("start-joinmultiplayer", e -> setCenter("start-joinmultiplayer-start", joinMultiPlayerGamePanel));
```

In my opinion this will create more mess. For now I just use _start-newmultiplayer_ and _start-joinmultiplayer_ for titles as well, as it is done for other buttons.

Commit:
Main menu - titles fix

Small fix to show proper titles for join multiplayer game and create
multiplayer game.
